src/content/merchants/index.md

```diff
@@ -17,6 +17,6 @@ Merchants, please note that:
 - If a merchant does not currently accept charge cards, they may consider contacting their financial institution to discuss accepting charge cards.
 - The transaction fees negotiated with the bank to accept cards from the private sector will also apply to government purchases.
 
-For more information on how to recognize GSA SmartPay card/account types and business lines, please view the [SmartTax Merchant Guide [PDF, 4 pages]](/files/smartpay-vendor-guide.pdf).
```

Comment:
Are we also deleting the PDF from Files, or does some other page still reference it outside the scope of this ticket?